README.md

```diff
@@ -61,7 +61,9 @@ You should generally derive these marker traits whenever possible.
 
 ###### Conversion Macros
 
-Zerocopy provides four macros for safe, zero-cost casting between types:
+Zerocopy provides six macros for safe casting between types. Unconditional
```

Comment:
Move the new sentence to immediately after the bulleted list, perhaps? It fits well with the sentence already there.

Reply:
Done.